tests/acceptance/30_generic_methods/log_rudder.cf

```diff
@@ -1,8 +1,8 @@
-#######################################################
+#########################################################################################
 #
-# Test checking if reporting is correcly done with rudder_logger
+# This will check reporting within log_rudder, by checking the defined classes
```

Comment:
Could you keep this test and make the class test a new one? It is good to keep a test of the reports output too.

Reply:
Damn - i overwrote by mistake the test :/ i'll correct this ASAP